docs/troubleshooting_android.md

```diff
@@ -0,0 +1,103 @@
+    * `TrackingConsent.GRANTED`: The SDK sends all current batched data and future data directly to the data collection endpoint.
+    * `TrackingConsent.NOT_GRANTED`: The SDK wipes all batched data and does not collect any future data.
+
+## Sample RUM sessions
+
+To control the data your application sends to Datadog RUM, you can secify a sampling rate for RUM sessions while [initializing the RumMonitor][] as a percentage between 0 and 100.
+
+```kotlin
+    val monitor = RumMonitor.Builder()
+            // Here 75% of the RUM Sessions will be sent to Datadog
+            .sampleRumSessions(75.0f)
+            .build()
+    GlobalRum.registerIfAbsent(monitor)
+```
+
+## Sending data when device is offline
+
+RUM ensures availability of data when your user device is offline. In cases of low-network areas, or when the device battery is too low, all the RUM events are first stored on the local device in batches. Each batch follows the intake specification. They are sent as soon as the network is available, and the battery is high enough to ensure the Datadog SDK does not impact the end user's experience. If the network is not available while your application is in the foreground, or if an upload of data fails, the batch is kept until it can be sent successfully.
+ 
+This means that even if users open your application while being offline, no data will be lost.
+ 
+**Note**: The data on the disk will automatically be discarded if it gets too old to ensure the SDK doesn't use too much disk space.
+
+## Migrating to 1.0.0
+
+If you've been using the former SDK (version `0.1.x` or `0.2.x`), there are some breaking changes
+introduced in version `1.0.0`, namely:
+
+### Logger.Builder
+
+{{< tabs >}}
+    {{% tab "After" %}}
+
+
+```java
+    Datadog.initialize(context, "my-api-key");
+
+    // …
+
+    logger = new Logger.Builder()
+            .setNetworkInfoEnabled(true)
+            .setServiceName("android-sample-java") // Sets the service name
+            .setLoggerName("my_logger") // Sets the logger name (within the service)
+            .setLogcatLogsEnabled(true)
+            .build();
+```
+
+{{% /tab %}}
+    {{% tab "Before" %}}
+
+
+```java
+    logger = new LoggerBuilder()
+        .withName("my-application-name") // This would set the service name
+        .withNetworkInfoLogging(this)
+        .build("my-api-key");
+```
+    {{% /tab %}}
+    {{< /tabs >}}
+
+
+### Attributes
+
+In earlier versions, attributes were created or removed with the `Logger.addField()` or `Logger.removeField()`
+methods. These methods were renamed for consistency purposes, and are now `Logger.addAttribute()`
+ and `Logger.removeAttribute()`. Their behavior remains the same.
+
+
+ ## Further Reading
+ <Link to Android common troubleshhoting issues: https://github.com/DataDog/dd-sdk-android/blob/master/docs/TROUBLESHOOTING.md>
```

Comment:
```suggestion
- [Common Android SDK troubleshooting issues][1]
[1]: https://github.com/DataDog/dd-sdk-android/blob/master/docs/TROUBLESHOOTING.md
```

but also, what is the difference between that troubleshooting page and this one? can we consolidate information?